chemprop/cli/convert.py

```diff
@@ -32,9 +32,12 @@ def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
 
     @classmethod
     def func(cls, args: Namespace):
-        args.output_path = args.output_path or Path(args.input_path.stem + "_v2.ckpt")
+        if args.output_path is None:
+            args.output_path = Path(args.input_path.stem + "_v2.ckpt")
         if args.output_path.suffix != ".ckpt":
-            args.output_path = Path(str(args.output_path) + ".ckpt")
+            raise ArgumentError(
+                argument=None, message=f"Output must be a `.ckpt` file. Got {str(args.output_path)}"
```

Comment:
```suggestion
                argument=None, message=f"Output must be a `.ckpt` file. Got {args.output_path}"
```

Reply:
thank you for pointing out that I don't need `str()`. I changed those.